coalib/bears/requirements/DistributionRequirement.py

```diff
@@ -0,0 +1,48 @@
+from coalib.bears.requirements.PackageRequirement import PackageRequirement
+import platform
+
+
+class DistributionRequirement(PackageRequirement):
+    """
+    This class is a subclass of ``PackageRequirement``, and helps specifying
+    distribution specific requirements, without using the manager name.
+    """
+
+    def __init__(self, manager, package, version=""):
+        """
+        Constructs a new ``DistributionRequirement``, using the
+        ``PackageRequirement`` constructor.
+
+        >>> dr =  DistributionRequirement('apt', 'libclang')
+        >>> dr.manager
+        'apt'
+        >>> dr.package
+        'libclang'
+
+        :param package: A string with the name of the package to be installed.
+        :param version: A version string. Leave empty to specify latest version.
+        """
+        PackageRequirement.__init__(self, manager, package, version)
```

Comment:
ahm shouldn't you like let the user give multiple package names for multiple managers and then use the righ one?

`def __init__(**manager_commands):` so the user can run like `DistributionRequirement(apt="something", dnf="else")`?